### PR TITLE
Backport cluster claiming deprecation.

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

Backport https://github.com/TheThingsIndustries/lorawan-stack/pull/3665

#### Changes

- Backport changes

#### Testing

Tested in Enterprise

##### Regressions

NA

#### Notes for Reviewers

@kschiffer: Please backport console related files (if any)

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: cmd/ttn-lw-cli/commands/end_devices.go [hunk 1]
```diff
@@ -1360,6 +1251,13 @@ This command may take end device identifiers from stdin.`,
 			if err := device.SetFields(dev, append(append(nsPaths, asPaths...), jsPaths...)...); err != nil {
 				return err
 			}
+
+			if device.ClaimAuthenticationCode.GetValue() == "" {
+				// Unset the field since qrcodegenerator.Generate does not support fieldmask
+				// and an empty field would result in a validation error.
+				device.ClaimAuthenticationCode = nil
+			}
+
```
Comment (KrishnaIyer) [inline cmd/ttn-lw-cli/commands/end_devices.go:1260]: This seems to be apparently never backported.

**Comment by kschiffer** [comment]
There is nothing to be backported for the Console since claiming was part of enterprise only. However, we can remove the claim tab for devices but I will do that in a separate PR.

**Review by adriansmares — APPROVED** [review]

**Review by nicholaspcr — COMMENTED** [review]
Everything looks good to me but the `CHANGELOG.md` doesn't include what has been removed

**Review by nicholaspcr — APPROVED** [review]
Approving since the only thing that I mentioned before is a simple fix and won't generate further discussions

**Comment by KrishnaIyer** [comment]
> Everything looks good to me but the CHANGELOG.md doesn't include what has been removed

This is already in Enterprise.

**Pull request merged**